demos/python_demos/llm_text_generation/converter.py

```diff
@@ -0,0 +1,305 @@
+#*****************************************************************************
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#*****************************************************************************
+
+# Based on: https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/254-llm-chatbot
+
+from functools import wraps
+import torch
+import openvino as ov
+from pathlib import Path
+from typing import Tuple
+import types
+
+def flattenize_inputs(inputs):
```

Comment:
```suggestion
def flatten_inputs(inputs):
```

Reply:
this is a copy from the original notebook